src/components/Home.js

```diff
@@ -1,20 +1,70 @@
-import React from 'react';
-import { NavLink } from 'react-router-dom';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useCollectionData } from 'react-firebase-hooks/firestore';
+import firebase from '../lib/firebase';
 import getToken from '../lib/tokens';
+import '../styles/Home.css';
+
+const db = firebase.firestore().collection('shopping_list');
```

Comment:
Once again I am making a suggestion for the two teams. This line of code is repeated in several files. I think it would be a better idea to transfer it to the `lib` directory where an instance of `firebase` is created so that we have something like:
```js
const shoppingListCollection = firebase.firestore().collection('shopping_list');
export { shoppingListCollection };
```
This avoids the repetition. Once again you don't need to implement it here.

Reply:
Great suggestion @nibble0101. I would second this suggestion. 👍